cicd/gitops/health-information-for-resources-deploment.adoc

```diff
@@ -0,0 +1,11 @@
+include::modules/gitops-document-attributes.adoc[]
+[id="health-information-for-resources-deploments"]
+= Monitoring health information for application reources and deployments
+:context: health-information-for-resources-deploment
+include::modules/common-attributes.adoc[]
+
+toc::[]
+
+The environment details page in the dev console displays the health status of the resources, such as routes, sync status, deployment config, deployment history.
```

Comment:
dev console is colloquial

Reply:
which resources? They need a specifying word, just "the resources" doesn't give enough information here.

Reply:
sync status is colloquial

Reply:
deployment config is colloquial